src/v2/view-builder/components/AuthenticatorOptions.js

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+import { ListView, loc } from 'okta';
+import skipAll from './views/SkipOptionalEnrollmentButton';
+import FactorRow from './views/FactorRow';
```

Comment:
the current mock is 
- for verification, https://oktawiki.atlassian.net/wiki/spaces/UX/pages/737092192/Sign+In+Widget+Design+-+IdX#Verification-(challenge)
- enrollment: https://oktawiki.atlassian.net/wiki/spaces/UX/pages/737092192/Sign+In+Widget+Design+-+IdX#Authenticator-enrollment

might be easier to have separated component for each (may end up with a little code redundancy) ?
or you can try to tweak `FactorRow` to meet different scenarios.